src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/VaultCertificateTests.ps1

```diff
@@ -0,0 +1,637 @@
hgj8pRNCb+DN3aDBEK8ksWwZ225BfVZxx7WyF8IRqmxHX2pDMlathbEee5lu3+mYNGhaqMQdkuyVBF2ISvWDqeZT8+zLx74CqmJVcisnhAb/2DjbceV57CJnFiCxg5BEtZ7uhCnNXLygkEZGVx3+N8BQp8h9u4/vtUT4mcKdOibxZ5E5WsgE3tZ8sw3JXS9Pq7RBu3JJ9SMnrmelR5iXi2Ux1RwkxWR3VAGNsyIt7jCs30FvVk3/cWxnCgzvaqVh+6aPlgSLRm25OwQJ1c1yO+HR2uGB46CHW9NRMZZ+CfM+8YMCphZYneUmoCnRCHgMMZYTDf92xlF5mdAmhrabws9tfqH77gdRrecq6vZDAWo84HA/NaGwQD+O30cD908lap4d196nBn10F53wElaQ2ITHjXICbCXcW2a2SguozhaJrBcEolm7bOIaMGfJ46PydKyMfiTCxasnGaqCbnape2wgoY+P/154BkdecpDkcc+07NusMqjoRIX+P4s1AAla9eXr5CTVSGX5kI2KjFV0FID2XFoTii4w+p6qKKYxZDSjRjez4Byj3IvUVrtZX0aBKfsd804HmQac4DCRLNwyJP9sbB4vOiS+kTqwtaLyi6mHsbWBAoIJaZ+YMHjNNTfIjf+SNZeljS4y7eHkHYwNj6Qni1pEi016XW72fSAuLPwLKDQSeryuf8opEntzvza9NGUAci1HDwE/TBR3Fg4Of9pY0v/6fyb35nkuzf8BNFebK0A5VG1oh4vNvTb76J1t2hcDzQk2NDvVSIgy65VrxDEJui355jec8UhoL8ZGUFKSNYYypTHZY+WmrGwdKqAz0SU7udCXC826oQQveNKC5gX/os+aPIB+bRrawvICY1+TZVKOBxN54ynVt0lMWcYF07WseckntfkVlNkFB7UU6pjV8WZW1NHsghE9Y3PgT/W8dwGQgs41EkQhbb3USyymYAFE0z3pd9m41u1fL1MZYF4AHS8/arW7WFW7FSodwEML8qPZEZS+LxppYKL08ynEvttXyWmcjxu0ceqvRcZKQJf0yhYXiXMxd4DqMn/gz4CowazUTl2qj6JA0sJSHuIxXX4Cioc4Hb4gxM00sHHSN98CBAdENfOD0vLNJDVqv+zfu2d4+Kp5nVvtVzwzYwTtV0DWGxg6P8drIzV3E/UL4Tv84WxsQwp2OpPlltoAq1lwqcztxJy3WnAzyvoBqTGG4xt9vKttHvP1/HFuQVDTjuIkA4ud/vvs6SeZelaZNCM4RhmqPtw0xsdugLHNXN07hPAL5DWQxKZQ7z2c7fPF8vREXkEEoFLu4EoI0Vhh8Oe+8GPpEF+oH+e1Js6hEfTEeoxnRctg1SuhkP48wp1d0DY8I33k2Xc6qwp7v2xqFzG1peHwLa5hfABGpzDysyYhbhWJg8oXMiHW8PsJWmP0Ua8VG3y1qeXxliczSabn8szHYd/jnttIX8awf/x3ZIWDBjVkdVW4mYLpmtWZ5+j76oWuJi7mkEb7OH+wYVRs1XAQ72W8Hf3DCo70UKSGqzRyv9ZNzn27koU9sI2Ib7cnN0jb58Z4PddLd5B2RCsUPTf7TNxf8N7IoxRlRW8BHJ7Foa5C/GxjeJ2fhE6kBWLSTpTVvnAGl+TSEmSvNJ5EcILA0zw12Dsiv0/SxCx6ZoWfT0RwQMMr7BiLmqhtHZmYXgeDYF8AViCW6alIQux2LpC+8b2jCbpAUObcQVk12MdIOi/EDVIVo03L5XQhbsCaQ06KUJfHMdeZkvwjjhvs6exEyxfCI35bCWWsOyy+hD2VTW+UnIpZFP344ToXQlIWUFZJ0bkCur9gw8OgoHyU6K8pL8b2tRFZTd6MPw1+Zyu4pDk2SWpt3dbtOutxaA7vk5pAuqHz4g2OznWwt4I0MpwJuLwINORbber/oO6JM1uR8Znkhh7l1RnYwZd4CfpPgHrTj44cZ5QLSlWWkXyMQ7eQZ7LWS0nC7vIvJ4L48q5h64l7184Yim60rCUdjEeAVcCokW2FrL3yYkNknf0nLGu/6AR61mvl4R2n59hSbytzM4XPiifXJ56DrZuvuqeg+IWvjqd0ygAUFxARqaIPCzS3hQatLovUMDJ3TTalBNL6hiT88s9oNBCf8VuT+Mhga7uomZTehyCWpCmXP1M+IMfeTecpl8CRq8Y+S3eXIKZhyj2yr6Rz2y4FljNx39h9qQ7PmVYL/VMPUTbv8NN/s2SbqJJSVS9rEg2l4k5mmIQ7xIANmmc22SxTjfJC8yAsv/goyreUsm+I8eToxGQ7GDk3OkyWWbKmI0/oEnmTiJfH8Vg0a05GKbsbhBzlenzzr0fdHDaO74b5TPez7RaIk2ez8SRweyiXSUzinz6Ud5a1xR2CIjwk+jCdnN2DDn0I71qhFKvrVU2kSOnK240owR/ziqK9DmtzBwZgLCrM8zWB905faLTRqpDrk4u/sIVbMzAVgyD0iKIaj+oFapG4UbrGKkUB5tJMmZh0Dx836BKqsFf6ID3XZcEIkfVWyWr47wMYzDZK/6cUzUtb3KfpczLQIWnqjLGAWkfsFR89EvJBsMxOeS1Pe2U8y3xtSp99aj8WPkJ0sG4aKsPLjZnCDsjXYGG864IVPw1QUaL9gbWQpAGWfmBYcbOnsZonQx5t5cm89akSDkCxeBUUNxnvWcBubZbQ88aCy6OVBnSArMrEmutz/nSA0GadvHbmU6BFnacrll+q7s8Fr0m2dpE/vSBaX1KbA8S+/uk69FA2hMqEtrHtYi/tI/ufo85sROv8W6sCQk1I3rW8WndcvD8CWWjqSry3rqUaRYH5D29PnmMdxYd6WTF2KbBLCERfMb1sUuN+7dVx3OremlTl8UArEhAxMIkJRtJWypyN8hzIQQC/bzo+FJXCRcfkG/GP2MpWOww/1fgFGcOZN2ARAQeySYwNzAfMAcGBSsOAwIaBBRy1LIXJAq1aMllupuwin+ROp2QqAQU3M+JiitCSxpAPnR3o6Z05jU+qqw="
```

Comment:
This will likely be flagged by CredScan , you will need to add an inline exception, after you've ensure that this cert is not used as an actual credential, but solely as an object for tests